app/models/rys_feature_record.rb

```diff
@@ -43,9 +43,29 @@ def self.request_store
     RequestStore.store[:rys_features_request_store]
   end
 
+  def self.hosting_request_store
+    if !RequestStore.store.has_key?(:rys_features_hosting_request_store)
+      RequestStore.store[:rys_features_hosting_request_store] = {}
+    end
+
+    RequestStore.store[:rys_features_hosting_request_store]
+  end
+
+  def self.hosting_active?(plugin)
```

Comment:
I don't like this solution. If you have plugin `pavel` with features `pavel.**.*` -> easy_hosting_service can just set

```ruby
Rys::Feature.add("pavel") do
  false
end